docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,284 @@
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+## Test procedure
+
+In this section and unless otherwise specified below, the term "[DNS Query]"
+follows the specification for DNS queries as specified in [DNS Query and
+Response Defaults]. The handling of the DNS responses on the DNS queries follow,
+unless otherwise specified below, what is specified for [DNS Response] in the
+same specification.
+
+1. Create a [DNS Query] with query type TXT and query name *Child Zone* ("TXT
+   query").
+
+2. Create two empty sets of strings, "SPF-Policies" and "SPFv1-Policies".
+
+3. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
+
+4. For each name server in *Name Server IP* do:
+
+   1. Send *TXT Query* to the name server and collect the [DNS Response].
+
+   2. Go to the next name server IP address if at least one of the following
+      criteria is met:
+
+      1. There is no DNS response.
+      2. [RCODE Name] of the response is not "NoError".
+      3. The AA flag is not set in the response.
+      4. There is no TXT resource record in the response.
+
+   3. If the name server responds with at least one TXT record, then, for each
+      TXT record:
+
+      1. If the record is an [SPF TXT record], then add the lowercased
+         [concatenation][concatenate] of all strings in the TXT record’s data to
+         the *SPF-Policies* set.
+
+   4. Go to the next name server.
+
+5. Compare the set of *SPF-Policies* retrieved from all name servers. If at
+   least one name server has responded with a set of SPF policies and a second
+   name server has responded with a different set of SPF policies, then output
+   *[Z11_INCONSISTENT_SPF_POLICIES]*.
+
+6. If no name server in the *Name Server IP* set responded with an authoritative
+   "NoError" response, then output *[Z11_UNABLE_TO_CHECK_SPF]* and terminate the
+   test.
+
+7. If the *SPF-Policies* set is empty, then output *[Z11_NO_SPF_FOUND]*
+   and terminate the test.
+
+8. For each string in *SPF-Policies* set, if the string is equal to `v=spf1` or
+   starts with `v=spf1` followed by a space, then add it to the *SPFv1-Policies*
+   set.
+
+9. If the *SPFv1-Policies* set is empty, then for each string in the
+   *SPF-Policies* set, output *[Z11_SPF_WRONG_VERSION]* with the version number
+   extracted from the string and the nameservers that returned the string.
+
+10. If the *SPFv1-Policies* set contains more than one element, then output
+    *[Z11_SPF1_MULTIPLE_RECORDS]* with the list of nameservers that returned
+    more than one SPF policy and terminate the test. The next steps assume
+    there is exactly one string in the *SPFv1-Policies* set ("SPF policy").
```

Comment:
I think it will be clearer if the last sentence is move to a bullet of its own in between 10 and 11.

"The next steps" --> "The following steps"?

Reply:
Fixing.